### PR TITLE
Server installation guide for Mac users added.

### PR DESCRIPTION
**Pull request by bjea**
Closes Issue #1090.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -324,3 +324,61 @@ To enable DEBUG on your docker server, call docker run with ``-e GA4GH_DEBUG=Tru
 This will set the environment variable which is read by config.py
 
 You can then get logs from the docker container by running ``docker logs (container)`` e.g. ``docker logs ga4gh_demo``
+
+-----------------------
+Deployment on Mac OS X
+-----------------------
+To deploy on Mac OS X, you need to install libraries and header code for `Python 2.7 <https://www.python.org/download/releases/2.7/>`_. It will be a lot easier if you have `Homebrew <http://brew.sh/index.html>`_, which is called the missing package manager for OS X, installed first. To install Homebrew, please paste the following at a Terminal prompt ($):
```
Comment (jeromekelleher) [inline docs/installation.rst]: Can you split these lines at ~80 chars please?

**Comment by jeromekelleher** [comment]
Looks good, +1 after the long lines have been split.

File: docs/installation.rst [hunk 2]
```diff
@@ -324,3 +324,122 @@ To enable DEBUG on your docker server, call docker run with ``-e GA4GH_DEBUG=Tru
...
+downloaded headers by pasting 
+``export C_INCLUDE_PATH="/usr/local/opt/openssl/include"`` into 
+``~/.profile`` file: 
+
+.. code-block:: bash
+
+  $ vim ~/.profile
+
+Click on "i" to switch to INSERT mode, paste
+``export C_INCLUDE_PATH="/usr/local/opt/openssl/include"``, click on "esc",
```
Comment (kozbo) [inline docs/installation.rst]: - Click on "i" to switch to INSERT mode, paste
+ vim instructions: type "i" to switch to INSERT mode, paste

**Comment by kozbo** [comment]
How about making this a separate document that is linked from the installation.rst file?

**Comment by david4096** [comment]
Closing in favor of #1279

**Pull request closed without merging**